docs/building-apps/wallet/intro.mdx

```diff
@@ -60,6 +66,19 @@ val walletCustom = Wallet(
 )
 ```
 
+```typescript
+const customClient: AxiosInstance = axios.create({
+  baseURL: "https://some-url.com/api",
+  timeout: 1000,
+  headers: { "X-Custom-Header": "foobar" },
+});
+let appConfig = new ApplicationConfiguration(DefaultSigner, customClient);
```

Comment:
Shouldn't DefaultSigner be there by default? (I.e. you don't need to set it manually)

Reply:
You don't, I can just pass `null` here

Reply:
Can you just omit it or it's not possible?